### PR TITLE
[Feature] Add atoms

### PR DESCRIPTION
**Pull request by titouanmathis**
## Added
- **Twig:** Add `Button` atom
- **Twig:** Add `Icon` atom
- **Twig:** Add blocks to `Accordion` and `Tabs` templates

## Changed
- **Doc:** replace Storybook by VitePress
- **JS:** Update components with the latest API from [@studiometa/js-toolkit](https://github.com/studiometa/js-toolkit)

### CODE REVIEW

File: packages/ui/atoms/Button/Button.twig [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+{# Default icon values #}
+{% set icon_position = icon_position|default('before') %}
+{% set icon_only = icon_only|default(false) %}
+
+{# Default attributes that could be overriden. #}
+{% set default_attributes = {
+  title: label,
+  type: tag == 'button' ? 'button' : false,
+  href: tag == 'a' and url is defined ? url : false,
```
Comment (perruche) [inline packages/ui/atoms/Button/Button.twig:27]: **suggestion**: 
Add `aria-label: icon_only ? label : false` to always have a clear name for icon buttons

File: packages/ui/atoms/Button/Button.twig [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+{% html_element tag with attributes %}
+  {% block content %}
+    {% if icon is defined and icon_position == 'before' %}
+      {% include '@ui/atoms/Icon/Icon.twig' with { name: icon, classes: { 'mr-3': not icon_only } } %}
+    {% endif %}
+    {% if icon_only %}
+      <span class="sr-only">{{ label }}</span>
+    {% else %}
+      {{ label }}
```
Comment (perruche) [inline packages/ui/atoms/Button/Button.twig:78]: **question**: Should label get by default an html element wrapper like `<span class="btn__label">` ?

File: packages/ui/molecules/Accordion/Accordion.twig [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<div data-component="Accordion">
```
Comment (perruche) [inline packages/ui/molecules/Accordion/Accordion.twig:1]: **issue/non-blocking**: Add comment for component api and blocks

**Review by perruche — APPROVED** [review]
This direction is great! Looking forward to use this!

File: packages/ui/molecules/Accordion/Accordion.twig (hunk 3 above)
Comment (titouanmathis) [reply packages/ui/molecules/Accordion/Accordion.twig:1]: There are other files missing this part, I would like to close this PR in order for `develop` to have the latest documentation.

File: packages/ui/atoms/Button/Button.twig (hunk 2 above)
Comment (perruche) [reply packages/ui/atoms/Button/Button.twig:80]: yeah, just having a `{% block label %}` will be enough to achieve this and keep things clean 👍

**Pull request merged**